### DISCUSSION TITLE
Can't upgrade Symfony to latest version

### DISCUSSION THREAD

**Discussion by NBlay** [symfony/symfony#48231] [Q&A]
Hi,

I have a project running on 5.4 and I want to update to the latest maintained version. I had issues because updating tried to reach for the old 'flex.symfony" server; but when I try to follow this guide https://symfony.com/blog/upgrade-flex-on-your-symfony-projects I get the following error.
"[Composer\Downloader\TransportException]                                                                                                            
  The "https://repo.packagist.org/packages.json" file could not be downloaded: php_network_getaddresses: getaddrinfo failed: No such host is known.   
  failed to open stream: php_network_getaddresses: getaddrinfo failed: No such host is known.  "

Any idea on how to fix this so I can upgrade ?

Thanks in advance,

**Comment by MatTheCat** [comment]
“No such host is known” implies an issue in DNS resolution or Internet access. Can you ping another domain from the system which ran composer?
